File: Classes/shape_area_and_perimeter/shapes.go
package main

import (
	"fmt"
)

type Rectangle struct {
  width float32;
  height float32;
}

func (r Rectangle) Area() float32 {
  area := r.width * r.height
  return area
}

type Circle struct {
  radius float32;
}

func (c Circle) Area() float32 {
  area := 3.14 * (c.radius * c.radius)
  return area
}

func main() {
	myRectangle := Rectangle { width: 10, height: 10 }
  fmt.Println(myRectangle.Area())
  myCircle := Circle { radius: 10 }
  fmt.Println(myCircle.Area())
}

File: Classes/shape_area_and_perimeter/shapes.py
class Rectangle:
  def __init__(self, width, length):
    self.width = width
    self.length = length

  def Area(self):
    return self.width * self.length

class Circle:
  def __init__(self, radius):
    self.radius = radius

  def Area(self):
    return 3.14 * (self.radius * self.radius)

myRectangle = Rectangle(10, 10)
print(myRectangle.Area())

myCircle = Circle(10)
print(myCircle.Area())
